whois-query/src/main/java/net/ripe/db/whois/query/pipeline/ExceptionHandler.java

```diff
@@ -43,6 +44,14 @@ public void exceptionCaught(final ChannelHandlerContext ctx, final ExceptionEven
             handleException(channel, Collections.singletonList(QueryMessages.inputTooLong()), QueryCompletionInfo.EXCEPTION);
         } else if (cause instanceof IOException) {
             handleException(channel, Collections.<Message>emptyList(), QueryCompletionInfo.EXCEPTION);
+        } else if (cause instanceof DataAccessException) {
+            LOGGER.error("Caught exception on channel id = {}, from = {} for query = {}\n{}",
+                    channel.getId(),
+                    ChannelUtil.getRemoteAddress(channel),
+                    query,
+                    cause.toString());
```

Comment:
The else clause (i.e. the existing code) logs the entire stack trace. 

In case of a DataAccessException, we don't care about the stack trace - just log that it's happened. This exception can happen if a database operation is interrupted on shutdown.